src/niquests/adapters.py

```diff
@@ -210,7 +211,7 @@ def send(
             data before giving up, as a float, or a :ref:`(connect timeout,
             read timeout) <timeouts>` tuple.
         :param verify: (optional) Either a boolean, in which case it controls whether we verify
-            the server's TLS certificate, or a string, in which case it must be a path
+            the server's TLS certificate, or a string/pathlib.Path, in which case it must be a path
```

Comment:
and maybe it’s worth describing it a little differently here?

Reply:
How about appending after "It is also possible to put the certificates (directly) in a string or bytes."

> we also accept PathLike objects

Whatever seems better to you.